tools/generate_docs/templates/r/cdap_streaming_program_run.md.tmpl

```diff
@@ -0,0 +1,17 @@
+{{template "header" .}}
+
+# Example
+
+```
+resource "cdap_streaming_program_run" "example" {
+  namespace = "staging"
+  type      = "spark"
```

Comment:
is it type or program?

Reply:
This is type. 
However I'm just reassigning the default here so removing this line.